### PR TITLE
Update DynamoDB Config table and fetcher Lambda to allow configs to b…

### PR DESCRIPTION
**Pull request by mengyangwang**
Summary
---------
1. added active config in collabs_example.json, default to N, which means should not be updated
2. modified fetcher.py to update collabarations if the active config is N

Test Plan
---------
tested locally
python3 -m hmalib.lambdas.fetcher
black hmalib
mypy hmalib

tested in aws:
go to dynamodb, delete table wangmengyangThreatExchangeConfig
terraform apply
make docker
make upload_docker
go to dynamodb  table wangmengyangThreatExchangeConfig, saw 'active' is 'N'

### CODE REVIEW

File: hasher-matcher-actioner/terraform/collabs_example.json [hunk 1]
```diff
@@ -9,6 +9,9 @@
             },
             "tags" : {
                 "L" : []
+            },
+            "active" : {
+                "S": "N"
```
Comment (jeberl) [inline hasher-matcher-actioner/terraform/collabs_example.json:14]: blocking: Can we make this type an AWS boolean value?https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBMapper.DataTypes.html

File: hasher-matcher-actioner/terraform/collabs_example.json [hunk 2]
```diff
@@ -9,6 +9,9 @@
             },
             "tags" : {
                 "L" : []
+            },
+            "active" : {
```
Comment (jeberl) [inline hasher-matcher-actioner/terraform/collabs_example.json:13]: Can we specify what `active` means? Maybe `fetcher_active` or `keep_in_sync`

**Review by jeberl — CHANGES_REQUESTED** [review]

File: hasher-matcher-actioner/terraform/collabs_example.json (hunk 1 above)
Comment (mengyangwang) [reply hasher-matcher-actioner/terraform/collabs_example.json:14]: sure will do

File: hasher-matcher-actioner/terraform/collabs_example.json (hunk 2 above)
Comment (mengyangwang) [reply hasher-matcher-actioner/terraform/collabs_example.json:13]: yeah I think fetcher_active makes sense

File: hasher-matcher-actioner/hmalib/lambdas/fetcher.py [hunk 3]
```diff
@@ -66,14 +66,17 @@ def lambda_handler(event, context):
...
+            ("#Name", "privacy_group", "tags", "fetcher_active")
+        ),
         ExpressionAttributeNames={"#Name": "Name"},
     )
 
     collabs = []
     for page in response_iterator:
         for item in page["Items"]:
-            collabs.append((item["Name"], item["privacy_group"]))
+            if item["fetcher_active"] == True:
```
Comment (jeberl) [inline hasher-matcher-actioner/hmalib/lambdas/fetcher.py:78]: nit: `if item["fetcher_active"] :`

**Review by jeberl — APPROVED** [review]
Looks good! Ship it

**Pull request merged**